faster_whisper/transcribe.py

```diff
@@ -231,7 +231,7 @@ def transcribe(
         word_timestamps: bool = False,
         prepend_punctuations: str = "\"'“¿([{-",
         append_punctuations: str = "\"'.。,，!！?？:：”)]}、",
-        vad_filter: bool = False,
+        vad_filter: Union[bool, Iterable] = False,
```

Comment:
Please update the doc string for "vad_filter" also.
In particular, I guess there is an expected format, like either [(start, end), ...] or [{"start": start, "end": end}]. Please describe this expected format somewhere.